api/main.py

```diff
@@ -49,6 +49,14 @@
                       user_scopes={"admin": "Superusers",
                                    "users": "Regular users"})
 pubsub = None  # pylint: disable=invalid-name
+API_VERSIONS = []
+
+
+@app.on_event('startup')
+async def init_api_versions():
+    """Startup event handler to initialize list of API versions"""
+    global API_VERSIONS
+    API_VERSIONS = ['latest', 'v0']
```

Comment:
Why is this done here rather than just directly in `API_VERSIONS` without a startup callback?